File: CS/WindowsApplication3/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.Skins;

namespace WindowsApplication3
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            Skin s = DevExpress.Skins.CommonSkins.GetSkin(wizardControl1.LookAndFeel);
            wizardControl1.Image = s[DevExpress.Skins.CommonSkins.SkinGroupPanelCaptionLeft].Image.GetImages().Images[0];
        }
    }
}

File: VB/WindowsApplication3/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Skins

Namespace WindowsApplication3
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim s As Skin = DevExpress.Skins.CommonSkins.GetSkin(wizardControl1.LookAndFeel)
			wizardControl1.Image = s(DevExpress.Skins.CommonSkins.SkinGroupPanelCaptionLeft).Image.GetImages().Images(0)
		End Sub
	End Class
End Namespace